yarn/check.py

```diff
@@ -222,7 +223,7 @@ def check(self, instance):
         # Get properties from conf file
         rm_address = instance.get('resourcemanager_uri', DEFAULT_RM_URI)
         app_tags = instance.get('application_tags', {})
-        filter_queues = instance.get('filter_queues', [])
+        queue_blacklist = instance.get('queue_blacklist', [])
```

Comment:
I find `filter_queues` a bit confusing because it doesn't tell if it's going to whitelist or blacklist the queues that are configured.

`queue_blacklist` is more explicit